### DISCUSSION TITLE
Any way to force pex to only include packages from my venv?

### DISCUSSION THREAD

**Discussion by mconigliaro** [pex-tool/pex#1498] [Q&A]
I just noticed that my pex file is way bigger than I expected, and when I started digging into the libraries I'm using, I discovered that botocore includes [this ~60MB data directory](https://github.com/boto/botocore/tree/develop/botocore/data) of what looks to be nothing but example responses from the AWS API. I manually deleted it from my virtualenv and my app kept working, so my idea was to just blow this directory away before running pex. I know this is unusual, but I was curious whether there was any way to force pex to only include packages from my venv so it only picks up my modified versions. I tried the `--no-pypi` and `--repo` options, but I couldn't get anything to work.

**Comment by jsirois** [comment]
There is not currently any way I know of. This issue, though, tracks the idea: #1361.
